pkg/controller/oneagent/oneagent_controller.go

```diff
@@ -252,6 +295,31 @@ func (r *ReconcileOneAgent) reconcileRollout(logger logr.Logger, instance *dynat
 	return updateCR, nil
 }
 
+func (r *ReconcileOneAgent) determineOneAgentPhase(instance *dynatracev1alpha1.OneAgent) (bool, error) {
+	// Check if this DaemonSet already exists
```

Comment:
nitpick : unnecessary comment on L#299 and L#309

Reply:
Removed comments